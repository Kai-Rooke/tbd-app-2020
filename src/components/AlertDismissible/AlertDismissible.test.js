import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import AlertDismissible from './AlertDismissible';
import { BrowserRouter as Router} from 'react-router-dom';

//cleans up 
afterEach(cleanup);

it("Renders AlertDismissible", () => {
    const { asFragment } = render(<Router><AlertDismissible/></Router>);
    expect(asFragment()).toMatchSnapshot();
});

/*it("Disclaimer Test Id", () => {
    const { getByTestId } = render(<Disclaimer/>);
    expect(getByTestId('ucpDisclaimer'));
});*/
