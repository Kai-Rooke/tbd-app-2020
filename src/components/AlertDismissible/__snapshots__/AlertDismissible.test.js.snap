// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders AlertDismissible 1`] = `
<DocumentFragment>
  <div
    class="fade Alert alert alert-dismissible show"
    role="alert"
  >
    <button
      class="close"
      type="button"
    >
      <span
        aria-hidden="true"
      >
        Ã—
      </span>
      <span
        class="sr-only"
      >
        Close alert
      </span>
    </button>
    <div
      class="alert-heading h4"
    >
      Please request GCSE grades
    </div>
  </div>
</DocumentFragment>
`;
